gammapy/cube/make.py

```diff
@@ -144,6 +150,9 @@ def _maps_sum_over_axes(maps, spectrum, keepdims):
         for name, map in maps.items():
             if name == "exposure":
                 map = _map_spectrum_weight(map, spectrum)
+            if name == "psf" or name == "edisp":
```

Comment:
OK, to omit this for now, but I guess in general we should reduce the `psf` map as well to only contain one energy bin...the `edisp` can be probably dropped in this case.  Maybe leave a TODO comment here?

Reply:
Added a todo